consensus/config.go

```diff
@@ -4,14 +4,15 @@ import "time"
 
 // timeout constant
 const (
-	// The duration of viewChangeTimeout; when a view change is initialized with v+1
-	// timeout will be equal to viewChangeDuration; if view change failed and start v+2
-	// timeout will be 2*viewChangeDuration; timeout of view change v+n is n*viewChangeDuration
-	viewChangeDuration time.Duration = 60 * time.Second
+	// default timeout configuration is shorten to 30 seconds as the consensus is 5s
+	viewChangeTimeout = 30
+	// The duration of viewChangeTimeout for each view change
+	viewChangeDuration time.Duration = viewChangeTimeout * time.Second
 
 	// timeout duration for announce/prepare/commit
-	phaseDuration     time.Duration = 60 * time.Second
-	bootstrapDuration time.Duration = 600 * time.Second
+	// shorten the duration from 60 to 30 seconds as the consensus is 5s
+	phaseDuration     time.Duration = 30 * time.Second
```

Comment:
Let's keep the timeout for consensus 60s? so the retries may save the consensus before we have to enter view change.

Reply:
we've shorten the consensus time from 8 to 5. Don't you think we can shorten the phaseDuration? The retry is faster as well, right now.

Reply:
I also want to speed up the converge process with shorter phaseDuration.

Reply:
The view change protocol is like the last resort. before that, it's better to rely on the retry logic because eventually the system's problems should only happen on the network level and retry should be able to get things through most of the time. We don't want to prematurely enter view change too quickly to make things more complicated. But it's reasonable to shorten the timeout a bit. Maybe 45 seconds?